docs/docs/firmware/state-monitor-esp32.md

```diff
@@ -100,19 +100,15 @@ A retained message will ensure the device auto-configures on startup.
 :::
 
 ### Recommended Configurations
-Below is a table showing possible **Connected Devices** and the supported `input types`. Check marks are indicating the recommended combinations to ensure intended behavior.
+Below is a table showing possible **Connected Device** and the supported `input types`. Check marks are indicating the recommended combinations to ensure intended behavior.

```

Comment:
The test in the comment here should be `connected devices`, but the header in the table should be `Connected Device`. Sorry for the confusion!